src/tool_provider/artifactory.rs

```diff
@@ -42,7 +40,7 @@ impl ToolProviderImpl for ArtifactoryProvider {
             .map_err(|error| ForemanError::ArtiAAError { error })?;
 
         if let Some(credentials) = tokens.get_credentials(host) {
-            builder = builder.header(ARTIFACTORY_API_KEY_HEADER, credentials.token.to_string());
+            builder = builder.header(AUTHORIZATION, format! {"Bearer {}", credentials.token});
```

Comment:
```suggestion
            builder = builder.header(AUTHORIZATION, format!("Bearer {}", credentials.token));
```
typo?

Reply:
Technically it's valid rust, but `()` probably makes more sense for function like macros.